extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeChannelInfoItemExtractor.java

```diff
@@ -0,0 +1,70 @@
+package org.schabi.newpipe.extractor.services.peertube.extractors;
+
+import com.grack.nanojson.JsonArray;
+import com.grack.nanojson.JsonObject;
+
+import org.schabi.newpipe.extractor.channel.ChannelExtractor;
+import org.schabi.newpipe.extractor.channel.ChannelInfoItemExtractor;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+
+import javax.annotation.Nonnull;
+
+public class PeertubeChannelInfoItemExtractor implements ChannelInfoItemExtractor {
```

Comment:
~~Isn't it possible that PeerTube channels have parent channels?~~ Yes but they are not shown in search results even by Peertube websites, so ignore this comment.